src/components/dashboard/AppCard.vue

```diff
@@ -168,11 +196,8 @@ watchEffect(async () => {
       </div>
     </td>
     <td class="w-1/4 p-2">
-      <div v-if="!isLoading && !props.channel" class="text-center">
-        {{ devicesNb }}
-      </div>
-      <div v-else class="text-center">
```

Comment:
here is the code who should stay

Reply:
For the shared thing as you shared above?